src/brogue/Recordings.c

```diff
@@ -1157,15 +1158,15 @@ void getDefaultFilePath(char *defaultPath, boolean gameOver) {
     }
 
     if (!gameOver) {
-        sprintf(defaultPath, "Saved #%s at depth %d", seed, rogue.depthLevel);
+        sprintf(defaultPath, "Saved %s #%s at depth %d", gameConst.versionString, seed, rogue.depthLevel);
```

Comment:
Note the addition of the versionString (e.g. `CE 1.12.0`) at the start of the recording filename. This allows the user the easily see when viewing a recording if the recording will be compatible with the currently selected variant. They will need to switch variant to the right variant and then use the view recording functionality. This is a very simple approach (rather than reading the save file, figuring out which variant was used, if we support it and then switching to that variant) but it is very understandable and good for a first version, I believe.

Reply:
I vote against this; since the variant info is in the header, the user should get a sensible error message?

Reply:
Try the behaviour as is, I think you'll see it's reasonable. If not, let me know.

Reply:
How about a separate folder per variant for saves, recordings, & high scores? User selects variant first, then can see high scores, saves, and recordings only for that variant. Personally, I have a lot of saves and recordings. I think folder separation is cleaner than filename.

Reply:
Also, there seems to be a bug related to attempting to open a save/recording for a different variant. An error message is displayed but after the message is cleared by clicking ok, the menu buttons aren't visible. The menu reappeared after I either hit the escape key or clicked on a random location.

Reply:
Folder separation is possible but needs to be slightly special-cased so that only variant data is stored in folders, or for the vanilla game we look in `bin/` as well, just to ensure old recordings are still reachable.

Reply:
What platform do you run on @zenzombie ? You seem to get different bugs and compiler warnings to me! I can't reproduce the above on Windows/WSL2/SDL

Reply:
Ok I did some more testing. Weird. It happens if I launch with the --wizard command line arg. It doesn't happen without that switch. I'm on Windows 10/SDL. IDE is VS Code (just switched recently).

Reply:
Ah, yes, I can reproduce with `--wizard` sometimes, I'll take a look. Suspect not a regression.